security/crowdsec/src/opnsense/mvc/app/controllers/OPNsense/CrowdSec/Api/GeneralController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: © 2021 CrowdSec <info@crowdsec.net>
```

Comment:
don't like these but we can work with it.. needs some glue to support your copyrights in LICENSE, but that file claims 2-Clause BSD

Reply:
I'm not sure i understand what you want me to do, but I can just remove the spdx stuff and leave a MIT license file in the root

Reply:
Nevermind. I have to think about it and edit later if needed. Shouldn't be a blocker :)